convert2rhel/actions/post_conversion/hostmetering.py

```diff
@@ -126,7 +126,7 @@ def _check_host_metering_configuration(self):
         :rtype: bool
         """
         if tool_opts.configure_host_metering is None:
-            logger.debug("You have not enabled configuration of host metering. Skipping it.")
+            logger.info("You have not enabled configuration of host metering. Skipping it.")
             return False
```

Comment:
Dots seems to be removed by accident?

Reply:
I'm not sure what you're referring to exactly here. In the line above I changed the debug log level to info because without the --debug option users wouldn't see this message that is more than just of debugging nature. No dots removed.